services/IoT/applications/libraries/python.md

```diff
@@ -87,13 +87,13 @@ The application configuration file must also be in the following format:
 {: #api_calls}
 
 
-Each method in the API client responds with either a valid JSON or boolean response in the case of success or IoTFCReSTException in the case of failure. The IoTFCReSTException contains the following properties, which an application can parse to get more information about the failure.
+Each method in the API client responds with either a valid JSON or boolean response in the case of success or IoTFCReSTException in the case of failure. The IoTFCReSTException contains the following properties, which an application can parse to get more information about the failure:
```

Comment:
IoTFCReSTException - need an article and identifier - the IoTFCReSTException response ?